app/scripts/controllers/replicaSet.js

```diff
@@ -445,7 +446,9 @@ angular.module('openshiftConsole')
             $scope.deploymentConfigDeploymentsInProgress[deploymentConfigName][rsName] = replicaSet;
           } else if (action === 'MODIFIED') {
             // After the deployment ends remove him from the deploymentConfigDeploymentsInProgress structure.
-            delete $scope.deploymentConfigDeploymentsInProgress[deploymentConfigName][rsName];
+            if($scope.deploymentConfigDeploymentsInProgress && $scope.deploymentConfigDeploymentsInProgress[deploymentConfigName]) {
```

Comment:
Yeah I was seeing the whole thing `undefined`.  Shall I update `$scope.deploymentConfigDeploymentsInProgress` to create an empty object outside the if statements?